acceptance/suites/tests/00_smoke/validate-syslog-logback.rb

```diff
@@ -50,15 +50,17 @@
   on(master, "service #{service_name} status")
 end
 
-
 step 'Backup logback'
   on(master, "mv #{logback_path} #{logback_backup}")
 
-step 'Modify logback configuration'
+step 'Modify logback configuration' do
   create_remote_file(master, logback_path, logback_config)
+  on(master, "chmod +r #{logback_path}")
+end
 
 step 'Restart puppetserver'
```

Comment:
Should there be a do / end block around this since there are multiple lines inside of it?